app/estimate_square_size_test.go

```diff
@@ -110,27 +110,22 @@ func Test_estimateTxShares(t *testing.T) {
 	type test struct {
 		name              string
 		squareSize        uint64
-		normalTxs         int
 		pfbCount, pfbSize int
 	}
 	tests := []test{
-		{"empty block", appconsts.DefaultMinSquareSize, 0, 0, 0},
-		{"one normal tx", appconsts.DefaultMinSquareSize, 1, 0, 0},
-		{"one small pfb small block", 4, 0, 1, 100},
-		{"one large pfb large block", appconsts.DefaultMaxSquareSize, 0, 1, 1000000},
-		{"one hundred large pfb large block", appconsts.DefaultMaxSquareSize, 0, 100, 100000},
-		{"one hundred large pfb medium block", appconsts.DefaultMaxSquareSize / 2, 100, 100, 100000},
-		{"mixed transactions large block", appconsts.DefaultMaxSquareSize, 100, 100, 100000},
-		{"mixed transactions large block 2", appconsts.DefaultMaxSquareSize, 1000, 1000, 10000},
-		{"mostly transactions large block", appconsts.DefaultMaxSquareSize, 10000, 1000, 100},
-		{"only small pfb large block", appconsts.DefaultMaxSquareSize, 0, 10000, 1},
-		{"only small pfb medium block", appconsts.DefaultMaxSquareSize / 2, 0, 10000, 1},
+		{"empty block", appconsts.DefaultMinSquareSize, 0, 0},
+		{"one small pfb small block", 4, 1, 100},
+		{"one large pfb large block", appconsts.DefaultMaxSquareSize, 1, 1_000_000},
```

Comment:
wait we can do 1_000_000 :exploding_head: since when!! wtf

Reply:
Your comment made me realize the name for this test is incorrect. Fixed in 5ba2204e80041f7ef5a8b4867239b43e9ffadbff